tools/testing/fetch_mailman.sh

```diff
@@ -26,3 +26,6 @@ commands_pre =
     pip uninstall -y falcon
     pip install $FALCON_ROOT
 EOT
+
+# NOTE(vytas): Patch a test that started failing around May, 2022.
+sed -i 's/def test_interact_default_banner/def skip_test_interact_default_banner/' src/mailman/utilities/tests/test_interact.py
```

Comment:
this is fine, but how about applying a patch?

Reply:
I'll reword to "Skip a test".